pkg/store/zookeeper.go

```diff
@@ -199,15 +203,22 @@ func (s *Zookeeper) AtomicDelete(key string, oldValue []byte, index uint64) (boo
 	return false, ErrNotImplemented
 }
 
-// Acquire the lock for "key"/"directory"
-func (s *Zookeeper) Acquire(path string, value []byte) (string, error) {
-	// lock := zk.NewLock(s.client, path, nil)
-	// locks[path] = lock
-	// lock.Lock()
-	return "", ErrNotImplemented
+// CreateLock returns a handle to a lock struct which can be used
+// to acquire and release the mutex.
+func (s *Zookeeper) CreateLock(key string, value []byte) (Locker, error) {
+	// FIXME: `value` is not being used since there is no support in zk.NewLock().
+	return &zookeeperLock{
```

Comment:
Self-comment: This is useless for now, I could just return the result of `NewLock` since it adheres with our `Locker` interface.

Initially, `Locker.Lock()` returned a channel as well to notify of lost lock (this is supported by Consul).
Removed for now for simplicity, will need to add back soon.